### ISSUE TITLE
omekasy-v1.3.0-x86_64-pc-windows-msvc.zip is broken

### ISSUE THREAD

**Issue by suzuki-shunsuke** [ikanago/omekasy#52]
https://github.com/ikanago/omekasy/releases/tag/v1.3.0

[omekasy-v1.3.0-x86_64-pc-windows-msvc.zip](https://github.com/ikanago/omekasy/releases/download/v1.3.0/omekasy-v1.3.0-x86_64-pc-windows-msvc.zip) can't be unarchived by `unzip`.

```console
$ unzip omekasy-v1.3.0-x86_64-pc-windows-msvc.zip 
Archive:  omekasy-v1.3.0-x86_64-pc-windows-msvc.zip
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of omekasy-v1.3.0-x86_64-pc-windows-msvc.zip or
        omekasy-v1.3.0-x86_64-pc-windows-msvc.zip.zip, and cannot find omekasy-v1.3.0-x86_64-pc-windows-msvc.zip.ZIP, period.

$ echo $?
9
```

**Comment by suzuki-shunsuke** [comment]
- https://github.com/ikanago/omekasy/pull/53

**Comment by suzuki-shunsuke** [comment]
The file extension is `.zip`, but the actual format is `.tar.gz`.

https://github.com/ikanago/omekasy/blob/14145afbb8bd679136e6d8f9219371cd98e07fcb/.github/workflows/release.yml#L66

I could unarchive the file using `tar` command.

```console
$ mv omekasy-v1.3.0-x86_64-pc-windows-msvc.zip omekasy-v1.3.0-x86_64-pc-windows-msvc.tar.gz
$ tar xvzf omekasy-v1.3.0-x86_64-pc-windows-msvc.tar.gz 
x archive/
x archive/LICENSE
x archive/omekasy.exe
x archive/README.md
```

**Comment by suzuki-shunsuke** [comment]
Thank you for merging #53 and releasing v1.3.1.
I've confirmed the issue was solved at v1.3.1.

```console
$ unzip omekasy-v1.3.1-x86_64-pc-windows-msvc.zip
Archive:  omekasy-v1.3.1-x86_64-pc-windows-msvc.zip
   creating: archive/
  inflating: archive/LICENSE         
  inflating: archive/omekasy.exe     
  inflating: archive/README.md
```
